pipes/WDL/workflows/create_enterics_qc_viz.wdl

```diff
@@ -20,21 +24,29 @@ workflow CreateEntericsQCViz {
 
         String?         grouping_column_name
         String?         output_filename
+
+        String?         custom_est_coverage_thresholds
+        String?         custom_contig_thresholds
+        String?         custom_assembly_thresholds
+        String?         custom_mean_q_thresholds
```

Comment:
Can we get rid of all workflow input variables that are only used once to boilerplate specify a task variable? Just add `meta { allowNestedInputs: true }` to the workflow and leave the task inputs unbound.

```suggestion
```